packages/components/slider/src/slider.vue

```diff
@@ -83,6 +83,7 @@
             :key="key"
             :mark="item.mark"
             :style="getStopStyle(item.position)"
+            @mousedown.stop="onSliderMarkerDown(item.point)"
```

Comment:
Why not use the `setPosition(item.position)`?

Reply:
You're right! I've updated the code to use setPosition.